engines/bops_api/app/services/bops_api/application/creation_service.rb

```diff
@@ -97,7 +97,10 @@ def send_email?(planning_application)
       def initialize_from_planning_application(planning_application)
         params_v2 = planning_application.params_v2 || raise_not_permitted_to_clone_error
 
-        @params = ActionController::Parameters.new(JSON.parse(params_v2))
+        # TODO not sure when this is needed: maybe only for test setup?
+        params_v2 = JSON.parse(params_v2) unless params_v2.respond_to?(:with_indifferent_access)
+
+        @params = ActionController::Parameters.new(params_v2)
```

Comment:
Do we need this to be `ActionController::Parameters`? Seems like it should be the same as submitted originally, i.e. a hash with indifferent access.